fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyzeStmt.java

```diff
@@ -73,144 +70,46 @@ public class AnalyzeStmt extends DdlStmt {
 
     private static final Predicate<Long> DESIRED_TASK_TIMEOUT_SEC = (v) -> v > 0L;
 
-    private final TableName optTableName;
+    public final boolean wholeTbl;
```

Comment:
remove this var, add a function `analyzeAllColumns`

Reply:
What is this method used to do?

Reply:
return CollectionUtils.isEmpty(optColumnNames);